src/core/component/functional/context/helpers.ts

```diff
@@ -0,0 +1,53 @@
+/*!
+ * V4Fire Client Core
+ * https://github.com/V4Fire/Client
+ *
+ * Released under the MIT license
+ * https://github.com/V4Fire/Client/blob/master/LICENSE
+ */
+
+const
+	minOnceEventNameLength = 'Once'.length + 'on'.length + 1,
```

Comment:
ИМХО, может просто записать сразу:


```suggestion
	minOnceEventNameLength = 7; // 'Once'.length + 'on'.length + 1,
```

Зачем нам создание лишних литералов и вычисления на импорте.

Reply:
Не согласен